### PR TITLE
Issue 15900: Add new fat project for Kerberos Ldap

### LINKED ISSUE #15900: KRB5: Add FAT project for Ldap Kerberos tests
For #12794

Add a new FAT project for Ldap Kerberos tests: `com.ibm.ws.security.wim.adapter.ldap_fat.krb5`

### PR DESCRIPTION
**Pull request by kristip17**
Fixes #15900

Start the FAT project skeleton -- no tests yet, will probably move the Beta fence test from #15640 to here.

### CODE REVIEW

**Comment by kristip17** [comment]
#build

File: .github/test-categories/SECURITY_5 [hunk 1]
```diff
@@ -1,3 +1,4 @@
 com.ibm.websphere.security_fat.1
 com.ibm.ws.security.auth.data_fat
 com.ibm.ws.security.client_fat
+com.ibm.ws.security.wim.adapter.ldap_fat.krb5
```
Comment (jvanhill) [inline .github/test-categories/SECURITY_5:4]: Can we put this in the same group as our regular LDAP fats? Might be more convenient.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
+
+fat.project: true
+
+tested.features:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd:19]: We can remove this.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	vmmapi-1.0
+
+-buildpath: \
+	commons-logging:commons-logging;version=1.1.3
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd:23]: I would maybe remove all these except for the LDAP dependency so we can start fresh.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/build.gradle [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+dependencies {
+  requiredLibs 'commons-logging:commons-logging:1.1.3',
+     project(':fattest.simplicity')
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/build.gradle:14]: maybe comment those out. I also think fattest.simplicity gets auto-added.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/FATSuite.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+                AlwaysPassesTest.class
+})
+public class FATSuite {
+
+    // Using the RepeatTests @ClassRule will cause all tests to be run three times.
+    // 1) without any modifications
+    // 2) [FULL mode only] again with all features upgraded to their EE8 equivalents
+    // 3) [FULL mode only] again with all features _and applications_ upgrade to Jakarta EE 9 equivalents
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/FATSuite.java:34]: Get rid of this. I don't think there is any value added to running with EE8 or EE9. We are testing the registry, not any EE components.

**Review by jvanhill — APPROVED** [review]
My findings are all nit piks. You can either address them now, or they will all probably get addressed naturally when you add real tests.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd (hunk 2 above)
Comment (kristip17) [reply dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd]: Done

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd (hunk 3 above)
Comment (kristip17) [reply dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd]: Done

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/FATSuite.java (hunk 5 above)
Comment (kristip17) [reply dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/FATSuite.java]: Done

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/build.gradle (hunk 4 above)
Comment (kristip17) [reply dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/build.gradle]: Removed

File: .github/test-categories/SECURITY_5 (hunk 1 above)
Comment (kristip17) [reply .github/test-categories/SECURITY_5]: Move to SECURITY_4

**Comment by kristip17** [comment]
#libby

**Pull request merged**